docs/content/configuration/global-configuration/command-line-arguments.md

```diff
@@ -7,34 +7,42 @@ toc: true
 docs: "DOCS-585"
 ---
 
+{{< custom-styles >}}
 
-The Ingress Controller supports several command-line arguments. Setting the arguments depends on how you install the Ingress Controller:
+<style>
+   h3 {
+     border-top: 1px solid #ccc;
+     padding-top:20px;
+   }
+</style>
 
-- If you're using *Kubernetes manifests* (Deployment or DaemonSet) to install the Ingress Controller, to set the command-line arguments, modify those manifests accordingly. See the [Installation with Manifests](/nginx-ingress-controller/installation/installation-with-manifests) doc.
-- If you're using *Helm* to install the Ingress Controller, modify the parameters of the Helm chart that correspond to the command-line arguments. See the [Installation with Helm](/nginx-ingress-controller/installation/installation-with-helm) doc.
+NGINX Ingress Controller supports several command-line arguments. Setting the arguments depends on how you install NGINX Ingress Controller:
+
+- If you're using *Kubernetes manifests* (Deployment or DaemonSet) to install NGINX Ingress Controller, to set the command-line arguments, modify those manifests accordingly. See the [Installation with Manifests](/nginx-ingress-controller/installation/installation-with-manifests) doc.
+- If you're using *Helm* to install NGINX Ingress Controller, modify the parameters of the Helm chart that correspond to the command-line arguments. See the [Installation with Helm](/nginx-ingress-controller/installation/installation-with-helm) doc.
 
 Below we describe the available command-line arguments:
-&nbsp;
+
 <a name="cmdoption-enable-snippets"></a>
 
 ### -enable-snippets
```

Comment:
It might be a good idea to use the hugo anchor format instead; these are not being applied within the page template. We'd have to test it out to see if the permalinks and the links used in the What's on this Page sidebar honor them. 

```md
### -enable-snippets {#enable-snippets}
```